File: 1877-minimize-maximum-pair-sum-in-array/1877-minimize-maximum-pair-sum-in-array.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        nums.sort()
        i=0
        j=len(nums)-1
        res=0
        while i<j:
            if nums[i]+nums[j]>res:
                res=nums[i]+nums[j]
            i+=1
            j-=1
        return res

File: 1877-minimize-maximum-pair-sum-in-array/1877-minimize-maximum-pair-sum-in-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minPairSum = function(nums) {
    nums.sort((num1,num2)=>num1-num2)
    let i=0
    let j=nums.length-1
    let temp=[]
    while(i<j){
        temp.push(nums[i]+nums[j])
        i++
        j--
    }
    return Math.max(...temp)
};